python/fixedlen_tagger.py

```diff
@@ -36,6 +36,10 @@ def __init__(self, syncword_tag, packetlen_tag, packet_len, stream_type):
         self.written = 0
         self.really_written = 0
 
+    def set_packet_len(self, packet_len):
+        self.packet_len = packet_len
+        self.stream.maxlen = self.packet_len
```

Comment:
This doesn't look right. According to the Python [documentation](https://docs.python.org/3/library/collections.html#deque-objects), the `maxlen` attribute of a deque is read-only. I don't know if there's a way to change the `maxlen` of a deque. Perhaps it's necessary to create a new deque and move the data.

Reply:
Ah yes, this doesn't work. Just read up on the documentation, pushing fix.